include/deal.II/base/patterns.h

```diff
@@ -661,6 +661,98 @@ namespace Patterns
   };
 
 
+
+  /**
+   * This pattern matches comma-separated values of arbitrary types. Each type
+   * has to match a pattern given to the constructor.
```

Comment:
maybe give a quick example of how it can be used and what is the parameter file, i.e.
```
set My properties = ....
```

Reply:
done.